### PR TITLE
ELB limitation, SNI

### PR DESCRIPTION
**Pull request by theist**
Add a limitation in the ELB section

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -855,6 +855,7 @@ Load Balancers
 -	🔸As an operator of a service behind an CLB or ALB, the latter phenomenon means you can also see puzzling or erroneous requests by clients of other companies. This is most common with clients using back-end APIs (since web browsers typically cache for a limited period).
 -	❗CLBs and ALBs take time to scale up, it does not handle sudden spikes in traffic well. Therefore, if you anticipate a spike, you need to “pre-warm” the load balancer by gradually sending an increasing amount of traffic.
 -	❗Tune your healthchecks carefully — if you are too aggressive about deciding when to remove an instance and conservative about adding it back into the pool, the service that your load balancer is fronting may become inaccessible for seconds or minutes at a time. Be extra careful about this when an autoscaler is configured to terminate instances that are marked as being unhealthy by a managed load balancer.
+-	❗CLB HTTPS listeners does not support SNI (Server Name Indication) if you want to workarround this you must either provide a certificate with as many SANs as you need or use TCP Listeners and end SSL ring in your instances.
```
Comment (ThanosBaskous) [inline README.md]: Phrasing suggestion: "CLB HTTPS listeners don't support Server Name Indication (SNI). If you need SNI, you can work around this limitation by either providing a certificate with Subject Alternative Names (SANs) or by using TCP listeners and terminating SSL at your backend."

**Comment by theist** [comment]
@ThanosBaskous looks good, I'm not native english writer so I usually accept any corrections to my phrasing

File: README.md [hunk 2]
```diff
@@ -855,6 +855,7 @@ Load Balancers
 -	🔸As an operator of a service behind an CLB or ALB, the latter phenomenon means you can also see puzzling or erroneous requests by clients of other companies. This is most common with clients using back-end APIs (since web browsers typically cache for a limited period).
 -	❗CLBs and ALBs take time to scale up, it does not handle sudden spikes in traffic well. Therefore, if you anticipate a spike, you need to “pre-warm” the load balancer by gradually sending an increasing amount of traffic.
 -	❗Tune your healthchecks carefully — if you are too aggressive about deciding when to remove an instance and conservative about adding it back into the pool, the service that your load balancer is fronting may become inaccessible for seconds or minutes at a time. Be extra careful about this when an autoscaler is configured to terminate instances that are marked as being unhealthy by a managed load balancer.
+-	❗CLB HTTPS listeners don't support Server Name Indication (SNI). If you need SNI, you can work around this limitation by either providing a certificate with Subject Alternative Names (SANs) or by using TCP listeners and terminating SSL at your backend..
```
Comment (ThanosBaskous) [inline README.md]: 2 periods at the end - can you remove one?

**Comment by ThanosBaskous** [comment]
Thank you @theist!

**Pull request merged**